README.md

```diff
@@ -0,0 +1,126 @@
+## GitHub Action Deployment (ghd)
+
+#### Requirements
+* poetry >= 1.X
+* python >= 3.8.X
+
+
+#### Personal GitHub Access Token
+To use `ghd` you need a personal GitHub access token. 
+To create this token, go to your profile settings and click on `Developer settings`.
+There you can find `Personal access tokens`. Create one with `repo_deployment` permissions.
+
+Export your created token to use ghd.
+```shell
+export GITHUB_TOKEN=YourTokenHere
+```
+For later useage you can set this to your local profile settings.
+
+#### Install dependencies
+First ensure that you have a running Poetry version.
+
+The pyproject.toml is located in the `docker` directory.
+```shell
+cd docker
+poetry install
+```
+
+To check if `ghd` is working, just run the following command:
+```shell
+poetry run ghd
+```
+
+You should see the following output:
+```shell
+Usage: ghd [OPTIONS] COMMAND [ARGS]...
+
+Options:
+  --help  Show this message and exit.
+
+Commands:
+  deploy     Create new deployment
+  inspect    Inspect deployment state history
+  list       List deployments
+  set-state  Set deployment state
+```
+
+#### Link ghd to PATH
+To use `ghd` from every location on your system, link the shell wrapper to your PATH.
+```shell
+echo 'export PATH="~/workspace/src/git/github/ghd/docker:$PATH"' >> ~/.profile
```

Comment:
```suggestion
echo 'export PATH="~/path/to/ghd/docker:$PATH"' >> ~/.profile
```

Reply:
Ups, danke für den Hinweis.

Reply:
I would also add here `ln -s $HOME/bin` tip - actually to my mind much better than litter `$PATH`.